index.html

```diff
@@ -11,272 +11,227 @@
+                <img
+                  src="images/medium.svg"
+                  alt="medium logo"
+                  class="social-svg"
+                />
+              </a>
+            </div>
+          </div>
+        </section>
+        <section class="my-works">
+          <h1 id="works-header">My Recent Works</h1>
+          <hr />
+          <div id="works-wrapper">
+            <img
+              id="works-placeholder"
+              src="images/Img Placeholder.png"
+              alt="works image Placeholder"
+            />
+            <article class="work-item" id="first-item">
+              <h1>Multi-Post Stories</h1>
+              <p class="inter">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a
+                standard dummy text.
+              </p>
+              <ul class="work-stack">
+                <li class="stack-item">css</li>
+                <li class="stack-item">html</li>
+                <li class="stack-item">bootstrap</li>
+                <li class="stack-item">Ruby</li>
+              </ul>
+              <a href="" class="btn-a w-part">See Project</a>
+            </article>
+            <article class="work-item">
+              <h1>Profesional Art Printing Data</h1>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <ul class="work-stack">
+                <li class="stack-item">html</li>
+                <li class="stack-item">bootstrap</li>
+                <li class="stack-item">Ruby</li>
+              </ul>
+              <a href="" class="btn-a">See Project</a>
+            </article>
+            <article class="work-item">
+              <h1>Profesional Art Printing Data</h1>
+              <p class="inter">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <ul class="work-stack">
+                <li class="stack-item">html</li>
+                <li class="stack-item">bootstrap</li>
+                <li class="stack-item">Ruby</li>
+              </ul>
+              <a href="" class="btn-a">See Project</a>
+            </article>
+            <article class="work-item">
+              <h1>Profesional Art Printing Data</h1>
+              <p class="inter">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <ul class="work-stack">
+                <li class="stack-item">html</li>
+                <li class="stack-item">bootstrap</li>
+                <li class="stack-item">Ruby</li>
```

Comment:
- [ ] Kindly add the right `font-family` to the technologies of the cards as shown in the Figma design 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/212331367-27445940-31ce-4bcd-aba9-6ef58bfca92a.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/212331463-5d4f7ff5-e42f-4268-a9a6-7e91a75417b0.png)